daemons/attrd/attrd_ipc.c

```diff
@@ -290,6 +292,13 @@ expand_regexes(xmlNode *xml, const char *attr, const char *value, const char *re
         regfree(r_patt);
         free(r_patt);
 
+        /* If we never matched anything, return REG_NOMATCH.  This should not be
+         * treated as an error by handle_regexes.
+         */
+        if (!matched) {
+            return REG_NOMATCH;
```

Comment:
This could overlap with our codes. Maybe pcmk_rc_op_unsatisfied or a new code

Reply:
Switched to pcmk_rc_op_unsatisfied and updated the code comment to explain things a little better.